### PR TITLE
Brexit continued

### PR DESCRIPTION
**Pull request by callumlocke**

### CODE REVIEW

**Comment by callumlocke** [comment]
Would @andygnewman or @i-like-robots be able to take a look at this?

File: components/card/index.js [hunk 1]
```diff
@@ -45,7 +50,7 @@ module.exports = React.createClass({
 								<ODate dateTime={item.publishedDate} />
 							</div>
 						</div>
-						<div className="n-card__meta">{actions}</div>
+						{actions}
```
Comment (callumlocke) [inline components/card/index.js]: removed the extra wrapper as it seems to be a mistake – double-nesting `n-card__meta` divs causes an o-grid-column nested inside another o-grid-column, doubling up gutters. 

also the 'official' template doesn't double-nest them: https://github.com/Financial-Times/next-card/blob/master/templates/card.html

File: components/stickies/sticky-quintet/_main.scss [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		width: (3/7) * 100%;
+		margin-right: -20px;
+	}
+
+	.sticky-quintet .sticky-item--4 .sticky-item__image {
+		display: block;
+		margin-bottom: 15px;
+	}
+
+	.sticky-item--5 .sticky-item__subheading { display: none; }
```
Comment (i-like-robots) [inline components/stickies/sticky-quintet/_main.scss]: visually hide so it stays accessible?

File: components/stickies/sticky-quintet/_main.scss [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+		&:after {
+			content: "";
+			display: table;
+			clear: both;
+		}
+	}
+
+	.sticky-quintet__others-pair-1 {
+		float: left;
+		width: (4/7) * 100%;
```
Comment (i-like-robots) [inline components/stickies/sticky-quintet/_main.scss]: `percentage(4 / 7)`

File: components/stickies/sticky-quintet/_main.scss (hunk 2 above)
Comment (callumlocke) [inline components/stickies/sticky-quintet/_main.scss]: do we have a preferred next mixin/technique for visual hiding?

Comment (i-like-robots) [inline components/stickies/sticky-quintet/_main.scss]: No mixin but this should be the main technique we use https://github.com/Financial-Times/n-util/blob/master/main.scss#L41-L50

File: bower.json [hunk 4]
```diff
@@ -16,8 +16,7 @@
     "n-message-prompts": "Financial-Times/n-message-prompts#^v2.0.0",
     "n-notification": "^2.0.0",
     "n-topic": "^5.0.0",
-    "n-barriers": "^1.0.0",
-    "o-colors": "^3.3.1"
```
Comment (matthew-andrews) [inline bower.json]: intentional?

File: components/stickies/index.js [hunk 5]
```diff
@@ -1,26 +1,27 @@
...
+
+		if (layoutHint === 'featured') {
+			if (items.length > 5) items.length = 5;
+
+			inner = <StickyQuintet items={items} />
+
+			// TODO: switch down to Quartet or Triplet or Singlet layout if it's fewer than 5 items
+			// (but for now, the Quintet layout can handle it by just leaving empty spaces)
 		}
+		else inner = <StickyList items={items} />
```
Comment (matthew-andrews) [inline components/stickies/index.js]: lol i didn't know you could do this.  javascript is nuts.

File: test/server/transforms/branding.test.js [hunk 6]
```diff
@@ -40,8 +41,8 @@ describe('Branding transform', () => {
 
 		xit('sets various branding properties?', () => {})
```
Comment (matthew-andrews) [inline test/server/transforms/branding.test.js]: `xit`?

File: bower.json (hunk 4 above)
Comment (matthew-andrews) [inline bower.json]: ah think you're using some speical next colour getting mixin instead,

**Comment by matthew-andrews** [comment]
:+1:

File: test/server/transforms/branding.test.js (hunk 6 above)
Comment (callumlocke) [inline test/server/transforms/branding.test.js]: no idea, that's in master though, not in this PR

Comment (matthew-andrews) [inline test/server/transforms/branding.test.js]: good point…  carry on

File: components/stickies/index.js (hunk 5 above)
Comment (i-like-robots) [inline components/stickies/index.js]: The linter should get angry at this for a lack of semi-colon. I guess when it sees JSX it just doesn't bother checking the line.

**Pull request merged**